test/appengine/test_gae_connectionpool.py

```diff
@@ -0,0 +1,193 @@
+import unittest
+import logging
+import sys
+
+from nose.plugins.skip import SkipTest
+
+try:
+    from google.appengine.api import urlfetch
+except ImportError:
+    raise SkipTest("App Engine SDK not available.")
```

Comment:
No, it's discovered by nose during the regular tests. I tried using nose-exclude to skip the app engine folder completely, but it seems nose-exclude doesn't work correctly w/ python3.